src/lib/Form/Type/FieldType/DateFieldType.php

```diff
@@ -21,7 +21,13 @@
  */
 class DateFieldType extends AbstractType
 {
-    private const EDIT_VIEWS = ['ibexa.content.draft.edit', 'ibexa.content.translate', 'ibexa.content.translate_with_location', 'ibexa.user.update'];
+    private const EDIT_VIEWS = [
+        'ibexa.content.draft.edit',
+        'ibexa.content.translate',
+        'ibexa.content.translate_with_location',
+        'ibexa.user.update',
+        'ibexa.product_catalog.product.edit',
```

Comment:
ibexa/content-forms shouldn't refer to symbols defined in ibexa/product-catalog.